package.json

```
@@ -60,8 +60,8 @@
     "eslint-plugin-simple-import-sort": "^12.1.1",
     "lucide-react": "^0.515.0",
     "prism-react-renderer": "^2.4.1",
-    "react": "^19.0.0",
-    "react-dom": "^19.0.0"
+    "react": "19.2.3",
+    "react-dom": "19.2.3"
```

Shouldn't it be pinned using `^` instead? (thus relaxed constraints)
```suggestion
    "react": "^19.2.3",
    "react-dom": "^19.2.3"
```

(same thing for `@types/react` at line 41)